src/zmq/zmqabstractnotifier.h

```diff
@@ -39,6 +41,7 @@ class CZMQAbstractNotifier
     void *psocket;
     std::string type;
     std::string address;
+    int outbound_message_high_water_mark; // aka SNDHWM
```

Comment:
`outbound_message_high_water_mark` not initialized in the constructor?

Reply:
thanks. it was set, just not by the constructor. i changed things around to address that. i also found that CZMQAbstractPublishNotifier.nSequence was not initialized, so i initialized that too now.